src/module-2/bootstrap-design/index.html

```diff
@@ -0,0 +1,618 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Main</title>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;500;600&display=swap"
+        rel="stylesheet">
+  <link crossorigin="anonymous" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/css/bootstrap.min.css"
+        integrity="sha384-F3w7mX95PdgyTmZZMECAngseQB83DfGTowi0iMjiWaeVhAn4FJkqJByhZMI3AhiU" rel="stylesheet">
+  <script crossorigin="anonymous"
+          integrity="sha384-/bQdsTh/da6pkI1MST/rWKFNjaCP5gBSY4sEBT38Q/9RBh9AH40zEOg7Hlq2THRZ"
+          src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/js/bootstrap.bundle.min.js"></script>
+</head>
+<body class="container-fluid bg-light">
+<header class="mb-4 row">
+  <div class=" d-flex flex-row align-items-center justify-content-start logo">
+    <img alt="Logo" class="m-4" src="images/logo.svg">
+    <h1 class="text-primary">Online Store</h1>
+  </div>
+  <nav>
+    <a class=" mx-2 text-decoration-none" href="">
+      <svg class="bi bi-house" fill="currentColor" height="16" viewBox="0 0 16 16" width="16"
+           xmlns="http://www.w3.org/2000/svg">
+        <path
+          d="M2 13.5V7h1v6.5a.5.5 0 0 0 .5.5h9a.5.5 0 0 0 .5-.5V7h1v6.5a1.5 1.5 0 0 1-1.5 1.5h-9A1.5 1.5 0 0 1 2 13.5zm11-11V6l-2-2V2.5a.5.5 0 0 1 .5-.5h1a.5.5 0 0 1 .5.5z"
+          fill-rule="evenodd"/>
+        <path
+          d="M7.293 1.5a1 1 0 0 1 1.414 0l6.647 6.646a.5.5 0 0 1-.708.708L8 2.207 1.354 8.854a.5.5 0 1 1-.708-.708L7.293 1.5z"
+          fill-rule="evenodd"/>
+      </svg>
+    </a>
+    <svg class=" mx-2 bi bi-chevron-double-right" fill="currentColor" height="16" viewBox="0 0 16 16"
+         width="16" xmlns="http://www.w3.org/2000/svg">
+      <path
+        d="M3.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L9.293 8 3.646 2.354a.5.5 0 0 1 0-.708z"
+        fill-rule="evenodd"/>
+      <path
+        d="M7.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L13.293 8 7.646 2.354a.5.5 0 0 1 0-.708z"
+        fill-rule="evenodd"/>
+    </svg>
+    <a class="text-decoration-none" href="">eCommerce</a>
+    <svg class=" mx-2 bi bi-chevron-double-right" fill="currentColor" height="16" viewBox="0 0 16 16"
+         width="16" xmlns="http://www.w3.org/2000/svg">
+      <path
+        d="M3.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L9.293 8 3.646 2.354a.5.5 0 0 1 0-.708z"
+        fill-rule="evenodd"/>
+      <path
+        d="M7.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L13.293 8 7.646 2.354a.5.5 0 0 1 0-.708z"
+        fill-rule="evenodd"/>
+    </svg>
+    <a class="mx-2 text-decoration-none" href="">Electronics</a>
+  </nav>
+</header>
+
+<div class="row">
+  <div class="col-3 filters d-flex flex-column gap-3">
+    <div class=" d-flex flex-row justify-content-between caption">
+      <h5 class="px-2 font-weight-lighter">Filters</h5>
```

Comment:
`h5` can be used _only_ if there are `h1`, `h2`, `h3` and `h4` already present on the page. Since there is no classic page structure with headers and content, you may use `h1` for the main header only and refuse other headings

Reply:
[Modified](https://github.com/front-camp-2021/html-css-module-Kyanka/commit/a96e851a8ea308063ab34f162ef68b4714e0db1e)